.github/workflows/ci.yml

```
@@ -100,41 +100,43 @@ jobs:
-    
-    # macOS向け単一ファイルパブリッシュ
-    - name: Publish macOS x64
-      run: dotnet publish src/fcode.fsproj -c Release -r osx-x64 --self-contained true -p:PublishSingleFile=true -o publish/osx-x64
-    
-    - name: Upload macOS artifact
-      uses: actions/upload-artifact@v4
-      with:
-        name: fcode-macos-x64
-        path: publish/osx-x64/fcode
\ No newline at end of file
+  # build-macos:
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)